src/cli/backup/sharepoint.go

```diff
@@ -128,11 +132,22 @@ func addSharePointCommands(cmd *cobra.Command) *cobra.Command {
 
 		fs.StringArrayVar(&site,
 			utils.SiteFN, nil,
-			"Backup SharePoint data by site ID; accepts '"+utils.Wildcard+"' to select all sites.")
+			"Select backup details by site ID; accepts '"+utils.Wildcard+"' to select all sites.")
 
 		fs.StringSliceVar(&weburl,
 			utils.WebURLFN, nil,
-			"Restore data by site webURL; accepts '"+utils.Wildcard+"' to select all sites.")
+			"Select backup data by site webURL; accepts '"+utils.Wildcard+"' to select all sites.")
+
+		fs.StringSliceVar(
+			&pageFolders,
+			utils.PageFN, nil,
+			"Select backup data by site ID; accepts '"+utils.Wildcard+"' to select all sites.")
+
+		fs.StringSliceVar(
+			&page,
+			utils.PageItemFN, nil,
+			"Select backup data by site page ID; accepts '"+utils.Wildcard+"' to select all pages within site.",
```

Comment:
we've been picky about forcing users to use IDs in the past.  Do sites have filenames that might make for a better use interface?

Reply:
You are correct. This isn't a great use case. Will switch to full page name (e.g. `home.aspx`).

Reply:
Resolved [8d70659](https://github.com/alcionai/corso/pull/2213/commits/8d70659e1e547e6e72dc8d58a618de112d6ba5d8)